oak/proto/storage.proto

```diff
@@ -18,7 +18,7 @@ syntax = "proto3";
 
 package oak;
 
-import "google/rpc/status.proto";
+import "google/grpc/status.proto";
```

Comment:
https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto suggests that `rpc` was the correct import path?

Reply:
Ah, this was from https://github.com/grpc/grpc/blob/master/src/proto/grpc/status/status.proto but the googleapis version seems to be upstream/canonical.  Fixed the first commit to reference googleapis and restored the path.